garden_ai/client.py

```diff
@@ -100,13 +108,23 @@ def __init__(
             GardenClient.scopes.resource_server
         )
 
+        self.funcx_client = self._make_funcx_client()
         self._set_up_mlflow_env()
 
     def _set_up_mlflow_env(self):
         os.environ["MLFLOW_TRACKING_TOKEN"] = self.garden_authorizer.access_token
         os.environ["MLFLOW_TRACKING_URI"] = GARDEN_ENDPOINT + "/mlflow"
         _request_header_provider_registry.register(BinaryContentTypeHeaderProvider)
 
+    def _make_funcx_client(self):
+        scope_to_authorizer = {
+            AuthScopes.openid: self.openid_authorizer,
+            SearchScopes.all: self.search_authorizer,
+            FuncXClient.FUNCX_SCOPE: self.funcx_authorizer,
+        }
+        funcx_login_manager = FuncXLoginManager(scope_to_authorizer)
+        return FuncXClient(login_manager=funcx_login_manager, do_version_check=False)
```

Comment:
`do_version_check=False` is a bit of a hack. Turning that off means that there are no outbound network calls on FuncXClient init, and thus nothing we need to mock right now. But if there's any pushback on this I'll go back and do the mocking.

Reply:
seems fine to me